src/components/ConsoleIO/ConsoleIO.tsx

```diff
@@ -0,0 +1,108 @@
+import { CONSOLE_DEFAULTS } from './ConsoleIO.constants';
+import { useConsoleIO } from './ConsoleIO.hook';
+import {
+  ConsoleContainer,
+  Frame,
+  TopTabs,
+  StyledTab,
+  ConsoleContent,
+  ScrollableTextField,
+} from './ConsoleIO.styles';
+
+import type { ConsoleIOProps } from './ConsoleIO.types';
+
+export const ConsoleIO: React.FC<ConsoleIOProps> = ({
```

Comment:
This file uses `React.FC<ConsoleIOProps>` but there is no `import React from 'react'` or `import type React` present. Depending on your TSX/JSX settings, using `React.FC` may require importing React types explicitly.

**Sugerencia:**
```typescript
Add `import React from 'react';` or `import type { FC } from 'react';` and then use `export const ConsoleIO: FC<ConsoleIOProps> = ({ ... }) => { ... }` or avoid `React.FC` and type props inline to remove the need for the React type import.
```